### ISSUE TITLE
Jellyfin doesn't fill the library

### ISSUE THREAD

**Issue by maku1958** [jellyfin/jellyfin#7123]
**Logs**
[2022-01-05 18:20:38.038 +00:00] [INF] [26] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing "Scanne Medien-Bibliothek"
[2022-01-05 18:20:38.040 +00:00] [INF] [26] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/video0"
[2022-01-05 18:20:38.041 +00:00] [INF] [26] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/home/vdr/Musik"
[2022-01-05 18:20:38.041 +00:00] [INF] [26] Emby.Server.Implementations.Library.LibraryManager: Validating media library
[2022-01-05 18:20:38.071 +00:00] [INF] [54] Emby.Server.Implementations.ScheduledTasks.TaskManager: "Scanne Medien-Bibliothek" Completed after 0 minute(s) and 0 seconds
[2022-01-05 18:20:38.072 +00:00] [INF] [54] Emby.Server.Implementations.ScheduledTasks.TaskManager: ExecuteQueuedTasks
[2022-01-05 18:20:38.112 +00:00] [INF] [17] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory /video0
[2022-01-05 18:20:38.113 +00:00] [INF] [60] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory /config/data/collections
[2022-01-05 18:20:38.113 +00:00] [INF] [71] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory /home/vdr/Musik -->

**Server System (please complete the following information):**
 - OS: [Docker on Ubuntu]
 - Jellyfin Version: [e.g. 10.7.7]
 - Hardware settings & device: [e.g. Operating System: Linux Architecture: X64]
 - Other hardware notes: [Media mounted on Server where Jellyfin-Docker runs.]
 
**Client System (please complete the following information):**
 - Device: [Motorala G30]
 - OS: [Android 11]
 - Client: [and Web/Browser]
 - Browser (if Web client): [Firefox]
 - Client and Browser Version: [Version 95.0.1 (64-bit)]

Here is my Problem.
No Library I have added will be filled with data. I want some day use LIveTV but first the recent recordings should be played back. The recordings where on EasyVDR in TS-Format.
The music is simple mp3, that shouldn't really be an issue.
Maybe the problem is that jellyfin-docker runs under ~$ and all the date is to find under /$ but the folders I added I could select in the dashboard.
I would really appreciate some Tips.
Manfred

**Comment by vgambier** [comment]
How big are the libraries you are trying to import?

**Comment by maku1958** [comment]
@vgambier 
Thanks for the reply.
The libraries include about 10000 songs and on series with 8 sequels and 12 shows.

Sorry I did not close this issue, but I have installed jellyfin in the old school way via apt install and used jellyfin:jellyfin for the music and video folders. It works fine now I am pleased. I have to mention that in my surrounding the use of one general user is well thought of. The server is not in the www and we are only two users. 

Maybe in some months, when I better understand what snap does and how to handle it  I change back to snap after an major server update.
I leave it to you to close that issue because I don't know if you still can reply when I close it.

Thanks again
Manfred
